server.ts

```diff
@@ -18,6 +18,14 @@ if (!DELPHI_READS_WEBHOOK_URL) throw new Error('"DELPHI_READS_WEBHOOK_URL" env v
 const app = express();
 const port = DELPHI_READS_BOT_PORT || 6000;
 
+const sectors: Sector[] = [
+  { slug: 'All Sectors', title: 'All Sectors' },
```

Comment:
on the FE and TG summary we called this `general`. might be nice to be consistent.

also, I think you need to map this to all the real sectors when populating `item.taxonomy`.